autoload/mundo.vim

```diff
@@ -409,12 +411,59 @@ function! s:MundoRefresh()"{{{
   call winrestview(winView)
 endfunction"}}}
 
+function! s:MundoUpdateCursor()"{{{
+    if !g:mundo_auto_preview
+        return s:MundoRefresh()
+    endif
+
+    if !get(g:, 'mundo_inline_undo_delay', 0)
+        call s:MundoRefresh()
+    endif
+
+    if get(g:, 'mundo_auto_preview_delay', 0) <= 0
+        return s:MundoPython('MundoRenderPreview()')
+    endif
+
+    call s:MundoRestartCursorTimer()
+endfunction"}}}
+
+function! s:MundoRestartCursorTimer()"{{{
+    call s:MundoStopCursorTimer()
+    let s:auto_preview_timer = timer_start(
+                \ get(g:, 'mundo_auto_preview_delay', 0),
+                \ function('s:MundoRenderPreviewDelayed'))
+endfunction"}}}
+
+function! s:MundoStopCursorTimer()"{{{
+    if s:auto_preview_timer != -1
+        call timer_stop(s:auto_preview_timer)
+        let s:auto_preview_timer = -1
+    endif
+endfunction"}}}
+
+function! s:MundoRenderPreviewDelayed(...)"{{{
+    if s:auto_preview_line == line('.')
```

Comment:
There should be some guards here, similar to the guards in the MundoRefresh() method. Two bugs I'm seeing while testing now are:
 * if you call `:MundoShow`, and then move back to your main editor `control-wl`, the cursor focus will go back to to the __mundo__ window when the timer fires. Focus should remain where it was (see the MundoRefresh method to see how that happens currently).
 * if you call `:MundoShow` and then call `:MundoHide`, the __mundo_preview__ window will reopen when the timer fires.